packages/block/examples/7002Requests.ts

```diff
@@ -21,7 +22,7 @@ const main = async () => {
   }
   const request = createWithdrawalRequest(withdrawalRequestData) as CLRequest<CLRequestType>
   const requests = [request]
-  const requestsRoot = await genRequestsTrieRoot(requests)
+  const requestsRoot = genRequestsRoot(requests, sha256)
```

Comment:
I  think this example can be deleted. 

If we want a minimal example, then the steps are:

1. Ensure the withdrawals contract is deployed
2. Create a tx which initiates a valid withdrawal
3. If Prague is active, then the requests will now contain the LOG of the above withdrawal

Reply:
This example can be found in `eip-7002.spec.ts`, and examples for 6110 and 7251 can be found in `eip-7002.spec.ts` and `eip-7002.spec.ts` in the `vm` package API tests (in the `EIPs` directory).